docs/direct_kernel_boot.md

```diff
@@ -0,0 +1,67 @@
+# Direct Kernel Boot
+
+Virtink uses Cloud Hypervisor as the underlying hypervisor for VMs. Cloud Hypervisor supports direct kernel boot into a `vmlinux` ELF kernel. It should work with a rootfs from most distributions and does not require an EFI system partition. To use the direct kernel boot feature in Virtink, both a kernel image and a rootfs volume is required.
```

Comment:
both a kernel image and a rootfs volume is required -> both a kernel image and a rootfs volume are required?